README.md

```diff
@@ -0,0 +1,701 @@
+
+```json
+{
+  //...
+  "expires": 1712310016, // one day later than to_timestamp
+  "action": "document/update",
+  "conditions": {
+    "document_ids": ["0A01.."],
+    "to_timestamp": 1712226632 // one day before expires
+  }
+}
+```
+
+By including an `"expires"` field which contains a timestamp one day ahead of the `"to_timestamp"` condition we are allowing operations within the capabilities scope to arrive up to one day late, but no longer. After `"expires"` has passed we can reject updates from authors depending on this capability, even if they would have fallen within the authorised scope. Correct configuration of this behaviour requires an understanding of the networks you will be using and the degree of delay you wish to tolerate. Misconfiguration has the potential to artificially introduce network partitions, where one peer accepted operations which another never received, however the capability used has now expired. This "deadlock" is expected to be resolved when new capabilities are issued, or if the operations are due for garbage collection in any case.
+
+### `from_timestamp`
+
+Including `from_timestamp` restricts a capabilities authority to only operations' which contain a greater `timestamp` in their header.
+
+```json
+{
+  //...
+  "action": "document/update",
+  "conditions": {
+    "document_ids": ["0A01.."],
+    "from_timestamp": 1712226632
+  }
+}
+```
+
+The above capability restricts authority to document `"0A01"` and only operations containing a timestamp `>` `1712226632`.
+
+### `to_seq`
+
+Including `to_seq` restricts a capabilities authority to only operations' which contain a lower `timestamp` in their header.
+
+An authors contributions to a document contain a monotonically increasing sequence number. By leveraging this property we can introduce a more strictly bounded authority than is possible when using timestamps.
+
+```json
+{
+  //...
+  "action": "document/update",
+  "conditions": {
+    "document_ids": ["0A01.."],
+    "to_seq": 100
+  }
+}
+```
+
+The above capability restricts authority to document `"0A01"` and only operations containing a sequence number `<` 100. Effectively this capability allows a maximum of 100 operations to be published to a document.
+
+### `from_seq`
+
+Including `from_seq` restricts a capabilities authority to only operations' which contain a greater `timestamp` in their header.
+
+### `not_before`
+
+The UTC Unix Timestamp in milliseconds specifying at which point this capability becomes valid.
```

Comment:
Proposal: don't use utc, use utc/1000 
This is used in DID contexts. It's fine enough granularity (you don't need ms acuracy) and consistently saves bytes?